src/qibocal/auto/execute.py

```diff
@@ -97,14 +97,14 @@ def current(self):
         assert self.head is not None
         return self.graph.task(self.head)
 
-    def run(self):
+    def run(self, qubits, platform, folder):
```

Comment:
Do not put them as argument, use attributes: in the end, it makes sense that the instantiated runner will run on a given platform, for some qubits, and write the result in a specified folder.

If ever you need to loop over something (e.g. you have a list of `(qubits, platform)`) you can always construct multiple executors (they are not memory heavy, they do not contain data).
```suggestion
    def run(self):
```